circleguard/gui.pyw

```diff
@@ -86,7 +86,7 @@ class MainWindow(QWidget):
 class MainTab(QWidget):
     def __init__(self):
         super(MainTab, self).__init__()
-        self.q = Queue()
+        self.queue = Queue()
```

Comment:
eh, I think self.q and cg are clear enough, though we don't use them much so being as succinct as possible is maybe not needed. I'm neutral on this one